### DISCUSSION TITLE
aiuto per i primi passi

### DISCUSSION THREAD

**Discussion by efestopini** [indomus/forum#2776] [HUB: Home Assistant]
Buongiorno a tutti, grazie ad un amico mi trovo con home assistant core (su docker) installato su di un minipc con ubunto server, io ne capisco poco o niente e non ho nozioni di programmazione, vorrei installare hacs seguendo la guida qui su indomus  

![screen indomus](https://github.com/indomus/forum/assets/154065991/968e9e64-8e04-4960-bada-1c3c8b3201c3)

naturalmente mi collego con putty, inserisco utente e password e do il primo comando, fa qualcosa in un secondo e si chiude putty, mi riconnetto, inserisco utente e password e do il secondo comando per riavviare il container ma da errore 

![screen terminale](https://github.com/indomus/forum/assets/154065991/b84330a9-4919-4ce9-affc-57686c4596e8)

essendo un riavvio del container ho pensato che riavviando il minipc si sarebbe riavviato di conseguenza il container, sono andato su aggiungi integrazione ma hacs non c'è, qualcuno riesce a capire dove sbaglio?

**Comment by efestopini** [comment]
ho installato l'interfaccia grafica per poter creare manualmente le cartelle mancanti e inserire il contenuto scaricato ed estratto, come da procedura manuale, naturalmente ho creato l'account github, creato il token ed editato il file di configurazione con le righe 
hacs:
  token: mio_token 
sostituendo "mio token" con il codice generato da github ma, in assistant da errore, non capisco proprio dove sbaglio

**Comment by indomus** [comment]
> Buongiorno a tutti, grazie ad un amico mi trovo con home assistant core (su docker) installato su di un minipc con ubunto server, io ne capisco poco o niente e non ho nozioni di programmazione, vorrei installare hacs seguendo la guida qui su indomus
> 
> ![screen indomus](https://private-user-images.githubusercontent.com/154065991/291065402-968e9e64-8e04-4960-bada-1c3c8b3201c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4ODY3ODIsIm5iZiI6MTcwMjg4NjQ4MiwicGF0aCI6Ii8xNTQwNjU5OTEvMjkxMDY1NDAyLTk2OGU5ZTY0LThlMDQtNDk2MC1iYWRhLTFjM2M4YjMyMDFjMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxOFQwODAxMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjhkNjQ2NDI2NGIzZTlmNjQ3YjZlODk5M2IxMTI4YTFkZTM3YTY4ZWRmYzAzNzA0YTNiNWY3Y2JhZWEwN2E4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uACxze7Mo7pw_UsPAbNFUJ2GW3gNQ8SX2Zi-YNOgIuo)

come leggi, la guida è espressamente per Linux Debian e non su Linux Ubuntu. E' palese che il container non sia in esecuzione, perché però non sappiamo dirtelo, tocca capire cos'hai fatto prima.

**Comment by efestopini** [comment]
Grazie della risposta, diciamo che questo mio amico ha voluto farmi un bello scherzo, mi ha installato tutto dicendomi di preparare il pc con ubuntu server perché migliore e più stabile e poi di seguire le guide qui su indomus, ora gli chiederò spiegazioni a lui
